frontend/packages/vkt/public/i18n/fi-FI/public.json

```diff
@@ -116,7 +116,17 @@
           },
           "partialExamsSelection": {
             "partialExamsTitle": "Valitse osakokeet",
-            "skillsTitle": "Valitse tutkinto (Tutkinnot voidaan suorittaa yhdessä tai erikseen)"
+            "speakingSkill": "Suullinen taito *",
+            "writingSkill": "Kirjallinen taito *",
+            "doFullExam": "Haluatko suorittaa koko tutkinnon? *",
+            "skillsTitle": "Valitse tutkinto (Tutkinnot voidaan suorittaa yhdessä tai erikseen) *",
+            "yesFullExam": "Kyllä (sis. suullinen taito, kirjallinen taito, ja ymmärtämisen taito)",
+            "noFullExam": "En, haluan suorittaa vain jonkin osakokeen"
```

Comment:
En alkanut tätä kummemmin vielä katselmoida mut nää vois laittaa ehkä järjestykseen tuota käännösexceliäkin varten. Ja toinen että `speakingSkill` ja `writingSkill` vois vaihtaa olemaan `oralSkill` ja `textualSkill`. Laitan nää tälläin exceliin.

Reply:
Tai ei itse asiassa tarvii järjestää uusiksi. Järjestetään joskus myöhemmin automaattisesti jos järjestetään.